<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keysForm</class>
 <widget class="QWidget" name="keysForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>172</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>172</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Keys</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="pubKeyLabel">
      <property name="text">
       <string>Public Key:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pubKeyButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pubKeyInput"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="privKeyInput"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="privKeyLabel">
      <property name="text">
       <string>Private Key:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="privKeyButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="keysGenButton">
      <property name="text">
       <string>Generate New Keys</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="keysUseButton">
      <property name="text">
       <string>Use This Keys</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="keysCancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
